monitor/grafana/zeebe.json

```diff
@@ -23630,6 +23630,409 @@
       "title": "Cluster",
       "type": "row"
     },
+    {
+      "collapsed": true,
+      "gridPos": {
+        "h": 1,
+        "w": 24,
+        "x": 0,
+        "y": 28
+      },
+      "id": 665,
+      "title": "Dynamic scaling",
```

Comment:
:thought_balloon: We could also move it to the "Cluster" section? That was my intention at least when I added it.

Reply:
:thinking: it make sense in general, but I was thinking that we may add more dashboard later on so one can only open this if needed. Need to think about it